README.md

```diff
@@ -28,6 +28,9 @@ http://entrproject.org/) utility:
 ```bash
 find . -name "*_shpec.sh" | entr shpec
 ```
+__Warning__: If you use `zsh` you should disable the `end` keyword. (to do that, run `disable -r end`.
```

Comment:
I really thing it belongs to the README, but indeed there it's noise.
(I'm not sure most user of zsh now about disabling keyword. (I discovered this very recently))

Maybe we should do a section *Shell Specificity*, where we put bash, zsh, ksh, or sh related nnotes.